examples/portfwd/portfwd.c

```diff
@@ -238,9 +238,12 @@ THREAD_RETURN WOLFSSH_THREAD portfwd_worker(void* args)
     int appFdSet = 0;
     struct timeval to;
     WOLFSSH_CHANNEL* fwdChannel = NULL;
-    byte buffer[4096];
-    word32 bufferSz = sizeof(buffer);
-    word32 bufferUsed = 0;
+    byte appBuffer[4096];
```

Comment:
Is it safe to assume 4096 is the max or is this just the size of the "chunks" being forwarded? Could this be an overridable define?

Reply:
It is arbitrary. I changed it to the same resettable constant name in echoserver.c.